Gemeinsamer Bibliotheksverbund ISBN.js

```diff
@@ -0,0 +1,238 @@
+				}];
+			}
+			//DDC is not the callNumber in Germany
+			item.callNumber = "";
+			item.complete();
+		});
+		translator.translate();
+
+	});
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"978-3-642-00230-4\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Eigentumsverfassung und Finanzkrise",
+				"creators": [
+					{
+						"firstName": "Otto",
+						"lastName": "Depenheuer",
+						"creatorType": "editor"
+					},
+					{
+						"lastName": "Symposion. Deutsche Stiftung Eigentum",
+						"fieldMode": true
+					}
+				],
+				"date": "2009",
+				"ISBN": "9783642002304",
+				"language": "ger",
+				"place": "Berlin ;Heidelberg",
+				"publisher": "Springer",
+				"series": "Bibliothek des Eigentums, Im Auftrag der Deutschen Stiftung Eigentum",
+				"seriesNumber": "7",
+				"attachments": [],
+				"tags": [
+					"Aufsatzsammlung",
+					"Aufsatzsammlung",
+					"Constitutional law",
+					"Constitutional law",
+					"Constitutional law",
+					"Eigentum",
+					"Finanzkrise",
+					"Haftung",
+					"Law",
+					"Law",
+					"Law",
+					"Online-Publikation",
+					"Online-Publikation",
+					"Ordnungspolitik"
+				],
+				"notes": [
+					{
+						"note": "\"Unter dem Thema 'Eigentumsverfassung und Finanzkrise' veranstaltete die Deutsche Stiftung Eigentum am 22. April 2009 in Berlin ein Symposion"
+					},
+					{
+						"note": "Die weltweite Finanzkrise ist Anlass, an Funktion und Wirkweise des privaten Eigentums in einer freiheitlichen Gesellschafts- und Wirtschaftsordnung zu erinnern. Privates Eigentum muss es geben, damit Verantwortung zugerechnet und Haftung realisiert, Gewinn und Verlust einem konkreten Verantwortungsträger persönlich zugerechnet werden können. Die Verletzung dieser konstitutiven Regeln einer auf privatem Eigentum basierenden Wirtschaftsordnung ist wesentlich ursächlich für das eingetretene Desaster auf den Finanzmärkten. Wie alle kulturellen Errungenschaften muss auch die Idee des privaten Eigentums, insbesondere die ihr immanente Bereitschaft zur Übernahme persönlicher Verantwortung des Eigentümers, jeder Generation erneut wieder in Erinnerung gerufen, überzeugend um sie geworben und vor allem vorbildhaft von den Akteuren in Politik und Wirtschaft vorgelebt werden. Nur so kann strukturelles Vertrauen in das Finanzsystem wieder gewonnen werden. Denn in ihrer vertrauensbildenden Kraft liegt die ordnungspolitische Funktion der Gewährleistung privaten Eigentums"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"3-86688-240-8\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Bilinguale Lexik nicht materieller lexikalischer Transfer als Folge der aktuellen russisch-deutschen Zweisprachigkeit",
+				"creators": [
+					{
+						"firstName": "Katrin",
+						"lastName": "Karl",
+						"creatorType": "author"
+					}
+				],
+				"date": "2012",
+				"ISBN": "9783866882416",
```

Comment:
The ISBN here doesn't match the query (even given the ISBN-10 to ISBN-13 transformation, since only the check digit should change). I can't really tell if that's the same exact book in a different format or the book is assigned more than one ISBN. http://m.lehmanns.de/shop/geisteswissenschaften/23253071-9783866882409-bilinguale-lexik vs http://www.worldcat.org/title/bilinguale-lexik-nicht-materieller-lexikalischer-transfer-als-folge-der-aktuellen-russisch-deutschen-zweisprachigkeit/oclc/874242034 In either case, the fuzzy matching is a bit concerning (unless it's actually very smart fuzzy matching and was supposed to happen). I can't really think of a reason why the end result should not contain the ISBN that was searched for, so we should be either not importing results that don't match the query, or substituting the ISBN with what was queried (though that seems risky).